app/src/main/java/com/imashnake/animite/features/searchbar/SearchFrontDrop.kt

```diff
@@ -108,9 +108,9 @@ fun SearchFrontDrop(
 
     Box(Modifier.fillMaxSize().drawBehind { drawRect(frontDropColor) })
 
-    searchList.data?.let {
+    if (!searchList.data.isNullOrEmpty()) {
         SearchList(
-            searchList = it,
+            searchList = searchList.data!!,
```

Comment:
This is one of the cases I was referring to earlier, where you might check `data` is not null, but that doesn't guarantee it hasn't changed
Basically, we can use `let` here to capture the value